### ISSUE TITLE
number of files/directories not shown correctly

### ISSUE THREAD

**Issue by benglitz** [Donders-Institute/dr-tools#9]
It would be more intuitive, if repocli first collected the count of all files/total amount of data to be transferred, and then correspondingly labelled the progressbar. Currently, one has little prediction of how long the upload will take before finishing.

**Comment by hurngchunlee** [comment]
@benglitz I wonder if you see improvement with this workaround.

**Comment by benglitz** [comment]
Yes, it has definitely improved! Thanks for implementing the workaround.

**Comment by hurngchunlee** [comment]
Thanks @benglitz I am closing this issue.
